indexer/tasks/src/multiera/multiera_address.rs

```diff
@@ -172,8 +182,18 @@ fn queue_certificate(
                 TxCredentialRelationValue::StakeDeregistration,
             );
         }
+        MultiEraCertificate::UnregCert(deregistration) => {
+            let credential = deregistration.stake_credential.to_cbor_bytes();
+
+            vkey_relation_map.add_relation(
+                tx_id,
+                &credential,
+                TxCredentialRelationValue::StakeDeregistration,
+            );
+        }
         MultiEraCertificate::PoolRegistration(registration) => {
-            let operator_credential = registration.pool_params.operator.to_raw_bytes().to_vec();
+            let operator_credential =
```

Comment:
this is unrelated, but I think this was incorrect, and it was incorrectly ported in the pallas to cml migration.